docroot/themes/contrib/civictheme/src/CivicthemeConfigManager.php

```diff
@@ -124,6 +129,11 @@ public function save($key, $value) {
     $config = $this->configFactory->getEditable("$theme_name.settings");
     $config->set($key, $value)->save();
 
+    // Set site slogan.
+    if ($key == 'components.site_slogan.content') {
```

Comment:
this condition should go before `$config->set()` so that the value would be only saved to `system.site` and not `components.site_slogan.content`

Reply:
@AlexSkrypnyk Updated as per your direction.